src/scss/semantic.scss

```diff
@@ -4750,36 +4750,6 @@ input::selection {
  */
 
 /*******************************
-             Flag
-*******************************/
-
-i.flag:not(.icon) {
-  display: inline-block;
-  width: 16px;
-  height: 11px;
-  line-height: 11px;
-  vertical-align: baseline;
-  margin: 0em 0.5em 0em 0em;
-  text-decoration: inherit;
-  speak: none;
-  font-smoothing: antialiased;
-  -webkit-backface-visibility: hidden;
-  backface-visibility: hidden;
-}
-
-/* Sprite */
-
-i.flag:not(.icon):before {
-  display: inline-block;
-  content: '';
-  background: url("themes/default/assets/images/flags.png") no-repeat -108px -1976px;
-  width: 16px;
-  height: 11px;
-}
-
-/* Flag Sprite Based On http://www.famfamfam.com/lab/icons/flags/ */
-
-/*******************************
```

Comment:
The `.flags` code seemed to be unused, and webpack was puking when trying to resolve the image.  Seemed safe to remove.  I can put this back if needed.  Same change was made in semantic-full.